index.js

```diff
@@ -191,7 +191,7 @@ module.exports = {
     "no-trailing-spaces": 0,
     "no-underscore-dangle": 0,
     "no-unneeded-ternary": 0,
-    "object-curly-spacing": 0,
+    "object-curly-spacing": [2, "always"],
```

Comment:
We were including this in a lot of projects using this. Makes the following an error:

```javascript
import {foo} from 'bar';
var obj = {'foo': 'bar'};
var obj = {'foo': 'bar' };
var obj = { baz: {'foo': 'qux'}, bar};
var obj = {baz: { 'foo': 'qux' }, bar};
```

see https://eslint.org/docs/rules/object-curly-spacing for more info

Reply:
I don't think `2` is valid here. I think it's just `["always"]`.
https://github.com/eslint/eslint/blob/master/lib/rules/object-curly-spacing.js#L24-L40

Reply:
![image](https://user-images.githubusercontent.com/1869157/35410889-6e9aa32c-01dc-11e8-83d9-8ff92581f799.png)

I'm pretty sure the rules are in the form of [level, options] where `level` is 1/warning or 2/error, and `options` can be anything

Reply:
Also, I linked the package to snapsheet-standard-web and tried running it with just passing 'always' and this is the result:

![image](https://user-images.githubusercontent.com/1869157/35411524-b1edca1c-01de-11e8-8e78-f8a8febc7119.png)